elasticache/redis/Acornfile

```diff
@@ -109,7 +109,7 @@ localData: info: """
 			 }
 			 containers: app: {
 					build: context: "./"
-					ports: publish: ["8080/http"]
+					ports: publish: ["5000/http"]
```

Comment:
Sangeetha suggested that this example port should match the port used in the example app